box-share-sdk/src/main/java/com/box/androidsdk/share/vm/DataWrapper.java

```diff
@@ -0,0 +1,55 @@
+package com.box.androidsdk.share.vm;
+
+import androidx.annotation.StringRes;
+
+/**
+ * A data wrapper to make sure LiveData changes can take network responses into account
+ * @param <T> the data type of the item in the response that will be returned
+ */
+public class DataWrapper<T> {
+    protected T mData;
```

Comment:
I'd put these in a constructor rather than have them as protected. And call from inheriting classes as super(,)

Reply:
Ok. I will change them to private and have them set from constructor instead.